src/components/organisms/AuthenticationModal/RegisterForm/RegisterForm.tsx

```diff
@@ -114,13 +114,13 @@ const RegisterForm: React.FunctionComponent<PropsType> = ({
           break;
       }
 
-      if (!result) return;
-
-      if (result.data === false) {
+      if (result?.data === false) {
```

Comment:
What if it results in `undefined`? The check will not pass and it will not throw an error

Reply:
It won't because of the typescript optional parameter `result?.data` if result is `null` or `undefined` this statement will fallback to `false`

Reply:
It will not fallback, since you have `===` without an implicit type conversion

Reply:
Actually I decided to remove the logic because this is the not correct place for it. Login and register don't have the required fields for the access check and this should be done in the venue landing page only.